src/Sylius/Bundle/CoreBundle/Fixture/Factory/ProductExampleFactory.php

```diff
@@ -356,10 +356,10 @@ private function createChannelPricings(ProductVariantInterface $productVariant,
      */
     private function createImages(ProductInterface $product, array $options)
     {
-        foreach ($options['images'] as $imageCode => $imagePath) {
+        foreach ($options['images'] as $imageType => $imagePath) {
             /** @var ImageInterface $productImage */
             $productImage = $this->productImageFactory->createNew();
-            $productImage->setCode($imageCode);
+            $productImage->setType($imageType);
```

Comment:
This way we can only add one image for given type (and cannot create images without type or two images with the same type).